types/node/http.d.ts

```diff
@@ -1546,7 +1546,43 @@ declare module "http" {
          */
         destroy(): void;
     }
-    const METHODS: string[];
+    type HTTPMethod =
```

Comment:
This list neither subsets nor supersets https://www.iana.org/assignments/http-methods/http-methods.xhtml , where is it from? I don't think this is actually a closed set.

Reply:
The list was the result of logging the methods array